dictionaries/telemetry/damageCauserName.json

```diff
@@ -13,11 +13,13 @@
   "BlackZoneController_Def_C": "Blackzone",
   "Boat_PG117_C": "PG-117",
   "BP_BRDM_C": "BRDM-2",
+  "BP_Dirtbike_C": "Dirt Bike"
```

Comment:
You're missing a couple of commas in each of the json files

Reply:
Fixed!